src/data/faq.json

```diff
@@ -484,7 +484,8 @@
                         "anchor": "other_platforms",
                         "active": true,
                         "textblock": [
-                            "It's not foreseeable at the moment if and when the app will be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification APIs. The community is invited to contribute and use the app for other APIs."
+                            "It's not foreseeable at the moment if and when the app will officially be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification Framework. The community is invited to contribute and use the app for other frameworks.",

```

Comment:
Could we please change "The app will use the Apple and Google Exposure Notification Framework."  to something which reflects that the app is already released?

Reply:
Suggest changing the historical statement from:

"It's not foreseeable at the moment if and when the app will officially be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification Framework."

to:

"The Corona-Warn-App uses the Apple and Google Exposure Notification Framework and the Robert Koch Institute (RKI) has correspondingly released the app to the Apple and Google app stores. The RKI does not currently plan to release  to other app stores."

Reply:
fixed (+ sentence rewording)

@MikeMcC399 I didn't use your recommendation for the reasons that F-Droid support has been neither confirmed nor denied, so the original statement is still valid.